internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -76,6 +84,46 @@ func (handler StorageHandler) addContext(next func(http.ResponseWriter, *http.Re
 	})
 }
 
+// processGetAsyncRequest is used to handle Async Get Requests
+func (handler StorageHandler) processGetAsyncRequest(writer http.ResponseWriter, request *http.Request) {
```

Comment:
How about changing the function name from processGetAsyncRequest to `processAsyncGetRequest`?

Reply:
Ok, I'll do that.